js/features/supportFileUploads.js

```diff
@@ -45,6 +45,13 @@ export function handleFileUpload(el, property, component, cleanup) {
 
     el.addEventListener('change', eventHandler)
 
+    // If the Livewire property has changed to null or an empty string, then reset the input
+    component.$wire.$watch(property, (value) => {
```

Comment:
I don't love adding a $watch here. It feels like hooking in to the reactivity engine could cause headaches in the future (lack of being cleaned up or something weird like that)

Are there other, more "stateless" approaches to this we can try?

Worst case we can hook into "commit" and check if inputs should be cleared when requests come back from the server.

But I feel like there might be something clever we're missing.